src/settings/settingsManager.ts

```diff
@@ -45,12 +46,26 @@ export class SettingsManager {
     }
   }
 
+  public getRootSlpPath(): string {
+    return this.appSettings.settings?.rootSlpPath as string;
```

Comment:
If you're needing to cast this, you're doing something wrong. You should actually handle the `.settings` case being undefined and not use optional chaining.